Makefile

```diff
@@ -58,3 +60,15 @@ prune:			## Remove containers and delete volume data.
 help:			## Show this help.
 	@echo "Make application docker images and manage containers using docker-compose files."
 	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m (default: help)\n\nTargets:\n"} /^[a-zA-Z_-]+:.*?##/ { printf "  \033[36m%-12s\033[0m %s\n", $$1, $$2 }' $(MAKEFILE_LIST)
+
+dev_build:			## Build all services.
+	${COMPOSE_PREFIX_CMD} docker-compose ${COMPOSE_DEV_ALL_FILES} build ${SERVICES_DEV}
+
+dev_up:				## Build and start all services.
+	${COMPOSE_PREFIX_CMD} docker-compose ${COMPOSE_DEV_ALL_FILES} up -d --build ${SERVICES_DEV}
+
+dev_down:			## Down all services.
+	${COMPOSE_PREFIX_CMD} docker-compose ${COMPOSE_DEV_ALL_FILES} down
+
+dev_logs:			## Tail all logs with -n 1000.
+	${COMPOSE_PREFIX_CMD} docker-compose $(COMPOSE_DEV_ALL_FILES) logs --follow --tail=1000 ${SERVICES_DEV}
```

Comment:
Please change everything from `docker-compose` to the 'new' syntax `docker compose`, see https://stackoverflow.com/a/66516826. I thought I changed it in some PR I sent to Yogesh's repo, but it's not here nor applied it seems.

Reply:
Found it: https://github.com/yogeshojha/rengine/pull/1161

Reply:
You can [cherry-pick](https://www.atlassian.com/git/tutorials/cherry-pick) that syntax modification commit into this branch. Then, we can merge this into master. 🙂 That way, you can modify that specific line of code, or you can backport it manually.

Reply:
Done